docs/guides/getting-started-java-spring.md

```diff
@@ -36,18 +36,22 @@ For example, in this guide we will outline a BPMN model to receive a payment req
 While stepping through this guide, you can visit our [sample repository](https://github.com/camunda/camunda-8-get-started-spring/blob/main/src/main/java/io/camunda/demo/process_payments/ChargeCreditCardWorker.java) with the completed code to check your work.
 :::
 
-<SmPrereqs/>
+[comment]: SmPrereqs/
 
-## Step 1: Install Camunda 8 Self-Managed
+## Prerequisites
 
-<Install/>
+Before getting started, ensure you have the following in your local environment:
 
-## Step 2: Create a new Spring Boot project
+- Access to your preferred code editor or IDE
+- [OpenJDK 21+](https://jdk.java.net/)
```

Comment:
Do we want to be opinionated about this link? I opted against including a specific link/recommendation in the C8Run documentation, as each user may have a different preference for install.

Reply:
I want to be more opinionated, but my first concern is this is the wrong link to OpenJDK. We want something like https://openjdk.org/install/

Reply:
The idea in providing a direct link was to make it easier for users to quickly access the right resource—the same link OpenJDK provides on their installation page. We also received user feedback requesting direct links for easier access. The link to https://openjdk.org/install/ works fine as well.

Reply:
I don't really love the OpenJDK install page - I think it's a little misleading/confusing, and OpenJDK offers no follow-up on setting JAVA_HOME when installed in this way. 

Using homebrew or apt-get for mac and linux installs is going to be a lot more straightforward, but this opens us up to needing to document multiple paths of entry because OpenJDK's documentation isn't terribly robust. I just want to make sure that being opinionated doesn't actually get us fielding more questions than it answers.

Reply:
Either way, a note to make sure JAVA_HOME is set correctly would probably help; that way, no matter the installation, that check happens before they continue.